scripts/conda/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+PACKAGE="param"
+
+python -m build -s .
+
+VERSION=$(python -c "import $PACKAGE; print($PACKAGE._version.__version__)")
```

Comment:
Out of curiosity, is there a reason this is not `PACKAGE.__version__`?

Reply:
This is to be sure that the automatic version of the file is generated. `param.__version__` is more "complex" with `try/except`.